### PR TITLE
#158558624 Base URL (Graphql) Endpoint Update

### PR DESCRIPTION
**Pull request by idrisadetunmbi**
#### What does this PR do?
- [x] Updates the endpoint for backend (Graphql) data.
- [x] Fixes some failing tests due to the update and some data changes from the backend.
- [x] Fixes a subtle bug with a previous feature.

#### Description of Task to be completed?
Base URL Endpoint update and code refactor

#### How should this be manually tested?
Clone the repo in the Android studio and run the app on an emulator.
Run tests with the available Gradle commands

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
#158558624

#### Screenshots (if appropriate)
NA

#### Questions:
NA

### CODE REVIEW

File: app/src/main/java/com/andela/mrm/room_information/resources_info/ResourcesInfoRepository.java [hunk 1]
```diff
@@ -31,7 +31,7 @@ public ResourcesInfoRepository(Context context, ApolloCall<RoomQuery.Data> apoll
 
     @Override
     public void loadRoom(final Callback callback) {
-        mRoomQuery.enqueue(new ApolloCall.Callback<RoomQuery.Data>() {
+        mRoomQuery.clone().enqueue(new ApolloCall.Callback<RoomQuery.Data>() {
```
Comment (TheDancerCodes) [inline app/src/main/java/com/andela/mrm/room_information/resources_info/ResourcesInfoRepository.java]: @idrisadetunmbi 
What does the `clone()` function do?

Comment (idrisadetunmbi) [reply app/src/main/java/com/andela/mrm/room_information/resources_info/ResourcesInfoRepository.java]: @TheDancerCodes 
It is meant to create a new instance of the `query` each time the `.enqueue` method is called. It is a bug fix from the previous feature. If the method gets called again (and it actually gets called if there is a failed request and the user tries to fetch the data again .e.g. by pressing the Retry button on a Snackbar), the `query` points to the previous instance without the `clone`, which results in an exception which crashes the app.

[Retrofit Illegal State Exception - SO](https://stackoverflow.com/questions/35093884/retrofit-illegalstateexception-already-executed)

Comment (TheDancerCodes) [reply app/src/main/java/com/andela/mrm/room_information/resources_info/ResourcesInfoRepository.java]: @idrisadetunmbi 
Great! Thanks for clarifying this. 👍

**Comment by TheDancerCodes** [comment]
LGTM.

I am merging this now.

**Pull request merged**